pandas/io/json/_json.py

```diff
@@ -645,7 +645,7 @@ def read_json(
 
     result = json_reader.read()
     if should_close:
-        filepath_or_buffer.close()
+        filepath_or_buffer.close()  # type: ignore
```

Comment:
same comments as above, this is very odd that you need to do it, whats the reveal_type?